### PR TITLE
Add a few more test pages 

### PR DESCRIPTION
**Pull request by Findus23**
I really like the idea of having one set of websites where every Matomo feature can be tested.

I added a few more examples:
- MediaAnalytics
- TagManager
- the opt-out form explained in the developer docs
- a simple opt-in
- using Klaro consent manager (it seems to be the most popular one, is FOSS and supports Matomo explicitly)

### CODE REVIEW

File: tagmanager/basic.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <meta charset="utf-8">
+    <title>Tag Manager Testpage</title>
+    <!-- Matomo Tag Manager -->
+    <script type="text/javascript">
+        var _mtm = window._mtm = window._mtm || [];
+        _mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
+        var d = document, g = d.createElement('script'), s = d.getElementsByTagName('script')[0];
+        g.type = 'text/javascript';
+        g.async = true;
+        g.src = 'https://dev.matomo/js/container_<?php echo $matomoTagManagerContainer;?>.js'
```
Comment (tsteur) [inline tagmanager/basic.php]: @Findus23 it uses the hard coded domain here

File: config/config.example.php [hunk 2]
```diff
@@ -1,3 +1,4 @@
 <?php
 $matomoUrl = 'http://localhost';
 $matomoIdSite = '1';
+$matomoTagManagerContainer = "ZNBuKNcy";
```
Comment (tsteur) [inline config/config.example.php:7]: could maybe add this to the documentation in https://github.com/matomo-org/test-examples/blob/main/README.md#configuration ?

Add the beginning of the file could actually add a comment like below?

```php
// WHEN ADDING NEW VARIABLES PLEASE DOCUMENT THEM IN THE README
```

File: tagmanager/basic.php [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        s.parentNode.insertBefore(g, s);
+    </script>
+    <!-- End Matomo Tag Manager -->
+</head>
+<body>
+<h1>Tag Manager Testpage</h1>
+<a href="?mtmPreviewMode=<?php echo $matomoTagManagerContainer;?>">Preview Mode</a>
```
Comment (tsteur) [inline tagmanager/basic.php]: could maybe add in brackets like `Preview Mode</a> (requires preview mode to be activated in Matomo)`?

**Review by tsteur — CHANGES_REQUESTED** [review]
Great @Findus23 very valuable 👍  left just a few minor comments

**Comment by diosmosis** [comment]
Merging this. If there are further changes that need to be made, they can always be done in another PR.

**Pull request merged**